scripts/SAMADhi_dbAnalysis.py

```diff
@@ -699,6 +700,23 @@ def force_symlink(file1, file2):
             os.remove(file2)
             os.symlink(file1, file2)
 
+def copyInconsistencies(basedir):
+    # try to read inconsistencies from previous job
+    # the file must be there and must contain the relevant data
+    try:
+        with open(basedir+'/data/DatasetsAnalysisReport.json') as jfile:
+            content = json.load(jfile)
+            return content["DatabaseInconsistencies"]
+    except IOError:
+        # no file. Return an empty string.
+        # This will happen if basedir is not (properly) set or if it is new.
```

Comment:
Since this seems not the expected case, it may be more clear to also print a message here.

Reply:
Ok, I added a message. You are right that in some case it may be unexpected (while it is totally if the script is run for the first time or without the -b option).